### PR TITLE
Feature5

### PR DESCRIPTION
**Pull request by m05e5**
# Blog app - Views
- Implemented the different views of the project.
- Implemented the routes to the different views
- add styling to the views

### CODE REVIEW

**Review by sinansevgi — COMMENTED** [review]
Hi @m05e5 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have a descriptive readme file.

Documenting your code is one of the core skills you need to learn as a junior developer. And The microverse projects are a really great place to test your skills and get feedback on them. I recommend that you document all your projects properly. Please add a descriptive readme file.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1 +1,25 @@
...
+  </a>
+  <% @posts.each do |post| %>
+    <a href="<%= @user.id %>/posts/<%= post.id %>" class="post">
+      <h2> <%= post.title %> </h2>
+      <div class='long-text'><%= post.text %></div>
+      <div class="counter"><p>Comments: <%= post.comment_counter %>,Likes: <%= post.like_counter %><p></div>
+    </a>
+  <% end %>
+   <a href="<%= @user.id %>/posts" class="see_more"> See all posts</a>
+</div>
```
Comment (DeliceLydia) [inline app/views/users/show.html.erb]: - [ ] When you are displaying a user page you should render only 3 recent posts not all the posts actually you should see all user posts when you click on the `see all posts` link please work on that

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -1 +1,25 @@
...
+    <div class='user-bio'>
+      <h3>Bio</h3>
+      <%= @user.bio %>
+    </div>
+  </a>
+  <% @posts.each do |post| %>
+    <a href="<%= @user.id %>/posts/<%= post.id %>" class="post">
+      <h2> <%= post.title %> </h2>
+      <div class='long-text'><%= post.text %></div>
+      <div class="counter"><p>Comments: <%= post.comment_counter %>,Likes: <%= post.like_counter %><p></div>
```
Comment (DeliceLydia) [inline app/views/users/show.html.erb]: - [ ] `Comments` and `likes` should be incremented when you add new ones check that as well when I visit a post being able to see comments but on the posts, itself comments and likes should display their numbers please work on that
like

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @m05e5, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AshakaE — APPROVED** [review]
### Hi @m05e5  🙋‍♀️, 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**